website/docs/_posts/2017-10-05-configuration-options.md

```diff
@@ -511,14 +511,16 @@ Set this to true if you wish to receive one PR for every separate major version
 
 ## node
 
-Configuration specific for all node.js version updates (e.g. `.travis.yml`).
+Configuration specific for all NodeJS version updates (e.g. `engines` field in `package.json`).
```

Comment:
Shouldn't it be Node.js?
![image](https://user-images.githubusercontent.com/6311784/37052649-33734c00-217a-11e8-89fd-825f38bec3e3.png)

Reply:
Fixed.

I tend to see `NodeJS`, or just `Node`, mentioned a lot more than `Node.js`, even though you're right that the trademark is `Node.js`.

So sticking with the trademark.